src/Faithlife.OAuth/Faithlife.OAuth.csproj

```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>0.1.1</Version>
+    <Version>0.2.0</Version>
```

Comment:
Could you change this to `<VersionPrefix>` and remove the one below? Only `<VersionPrefix>` should be specified.